.github/workflows/coverage-report.yml

```
@@ -79,15 +79,15 @@ jobs:
     steps:
       # We need to re-download the artifact in each matrix job
       - name: Download Individual Coverage Reports
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
         with:
           name: individual-coverage-data # Match upload name
           path: coverage-artifacts
           github-token: ${{ secrets.GITHUB_TOKEN }}
           run-id: ${{ needs.report.outputs.run_id || github.event.workflow_run.id }} # Use run_id from needs if possible
```

actionlint (1.7.12)

[error] property "run_id" is not defined in object type {matrix: string}

(expression)